package.json

```diff
@@ -27,6 +27,14 @@
       "script": "./dist/jquery.slim.min.js",
       "default": "./dist-module/jquery.slim.module.min.js"
     },
+    "./factory": {
+      "node": "./dist/jquery.factory.js",
+      "default": "./dist-module/jquery.factory.module.js"
```

Comment:
I actually wonder if we need the ESM version here at all, since this is only useful for Node.js...